### PR TITLE
SRVKP-6898: Replace font awesome with custom css in dynamic plugin #204

### PR DESCRIPTION
**Pull request by lokanandaprabhu**
**Description:**
Changed co-spin classname to co-spin-pipelines to aviod conflict with console class name

### CODE REVIEW

File: src/components/styles/common.scss [hunk 1]
```diff
@@ -7,12 +7,10 @@
   }
 }
 
-.co-spin {
-  animation: co-spin 2s infinite linear;
-}
+.co-spin-pipelines {
```
Comment (spadgett) [inline src/components/styles/common.scss:10]: I'd avoid the `co-` prefix altogether in your plugin. Then you never risk conflicting with console classes that always use the prefix. Something like

```suggestion
.pipelines-console-plugin__spin {
```

would be better imo. See the recommendations here:

https://docs.openshift.com/container-platform/4.17/web_console/dynamic-plugin/overview-dynamic-plugin.html#general-plug-in-guidelines

Comment (lokanandaprabhu) [reply src/components/styles/common.scss]: Updated the PR. PTAL.

**Review by spadgett — APPROVED** [review]
/lgtm

Thanks 👍

**Review by vikram-raj — COMMENTED** [review]
/lgtm

**Pull request merged**